python/py-pytzdata/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pytzdata
+version             2020.1
+revision            0
+license             MIT
+platforms           darwin
+supported_archs     noarch
+maintainers         {@xeron gmail.com:xeron.oskom} openmaintainer
+description         The Olson timezone database for Python
+long_description    ${description}
+
+python.versions     27 36 37 38 39
+
+homepage            https://pypi.python.org/pypi/${python.rootname}/
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
```

Comment:
`distaname` and `master_sites` are the defaults from the python PG; please remove